jobqueue_features/tests/test_mpi_wrapper.py

```diff
@@ -92,34 +100,82 @@ def test_which(self):
         # Check it returns None when a file is not executable
         self.assertIsNone(which(os.path.realpath(__file__)))
 
+    def test_mpi_wrap_execution(self):
+        # Only check the ones that work in CI
+        for launcher in [MPIEXEC, OPENMPI]:
+            # Include some (non-standard) OpenMPI options so that we can run this in CI
+            if not self.is_mpich():
+                self.launcher_args = "--allow-run-as-root --oversubscribe"
+            else:
+                self.launcher_args = ""
+            if which(launcher["launcher"]) is not None:
```

Comment:
If we had a better way of figuring out whether a launcher is supported/installed, we could loop over `SUPPORTED_MPI_LAUNCHERS`  in this `for` loop